python/pyspark/ml/classification.py

```diff
@@ -154,7 +157,10 @@ def setThreshold(self, value):
     @since("1.4.0")
     def getThreshold(self):
         """
-        Gets the value of threshold or its default value.
+        Gets the value of threshold or attempt to convert thresholds to threshold if set, or default
```

Comment:
I just noticed that both this and Scala doc is inaccurate.

Scala side says:
```
   * Get threshold for binary classification.
   *
   * If [[threshold]] is set, returns that value.
   * Otherwise, if [[thresholds]] is set with length 2 (i.e., binary classification),
   * this returns the equivalent threshold: {{{1 / (1 + thresholds(0) / thresholds(1))}}}.
   * Otherwise, returns [[threshold]] default value.
```

But actually, the logic is "if thresholds is set and is length 2, return 1 / (1 + t(0) / t(1) ). Otherwise return threshold or its default value."

Seems to me we should update both Scala and Python doc to reflect this.